documentation/egress.md

```diff
@@ -15,6 +15,37 @@ Egress providers use [operations](./api/operations.md) to provide status.
 
 The `--no-http-egress` flag requires users to specify an egress provider by preventing the default HTTP response for logs, traces, dumps, gcdumps, and live metrics.
 
+## Egress Extensibility (8.0+)
```

Comment:
I think this should go to the bottom of the page, below the `Examples of Egressing a dump to blob storage` section; concerns about egress extensibility will be a very small slice of .NET Monitor users and we shouldn't force everyone to attempt to understand it and then realize they generally don't need to know about it.